src/main/java/com/codeborne/selenide/webdriver/WebDriverFactory.java

```diff
@@ -77,6 +77,10 @@ public WebDriver createWebDriver(Config config, @Nullable Proxy proxy, @Nullable
   }
 
   private void setLoadTimeout(Config config, WebDriver webdriver) {
+    if (config.pageLoadTimeout() < 0) {
+      log.warn("Page load timeout is not changed because pageLoadTimeout is negative integer: {} ms", config.pageLoadTimeout());
```

Comment:
Do we really need to log these warnings?
Wasn't the goal to avoid warnings in logs?

Reply:
Let's remove it.